tests/common/test_transaction.py

```diff
@@ -715,10 +699,6 @@ def test_validate_fulfillments_of_transfer_tx_with_invalid_params(transfer_tx,
     with raises(TypeError):
         transfer_tx.operation = "Operation that doesn't exist"
         transfer_tx.fulfillments_valid([utx.conditions[0]])
-    with raises(ValueError):
```

Comment:
This was testing the case where you enforced the number of fulfillments to be the same as the number of conditions on a transaction. That is no longer true.

The only case where this raises a `TypeError` is when the number of fulfillments is different then the number of `input_condition_uris` which is tested with `transfer_tx.fulfillments_valid([])`